.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: • 🛫 Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  get-available-images-matrix:
+    uses: ./.github/workflows/__shared-get-available-images-matrix.yml
+
+  prepare-images-to-build:
+    needs: [get-available-images-matrix]
+    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        image: ${{ fromJson(needs.get-available-images-matrix.outputs.images-matrix) }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: get-latest-tag-sha
+        uses: actions/github-script@v6.4.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const tags = await github.paginate(github.rest.repos.listTags, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            });
+
+            const tagFilter = '${{ matrix.image }}-';
+            const imageTags = tags.filter(tag => tag.name.startsWith(tagFilter));
+
+            // Sort tags regarding semver
+            imageTags.sort((a, b) => {
+              const aVersion = a.name.replace(tagFilter, '');
+              const bVersion = b.name.replace(tagFilter, '');
+              return bVersion.localeCompare(aVersion, undefined, { numeric: true });
+            });
+
+            core.setOutput('sha', imageTags.length ? imageTags[0].commit.sha : '');
+
+      - id: should-build-image
+        uses: ./.github/actions/should-build-image
+        with:
+          base-sha: ${{ steps.get-latest-tag-sha.outputs.sha }}
+          build-context: images/${{ matrix.image }}
+
+      - id: release
+        if: steps.should-build-image.outputs.should-build-image == 'true'
+        # FIXME: wait for PR to be merged https://github.com/release-drafter/release-drafter/pull/1255
+        uses: neilime/release-drafter@fix/strip-prefix-before-comparing-version
```

Comment:
This fixme can be fixed as a new version of release drafter is available

Reply:
Good just as a note this is cut and paste from ac repo so it might be applicable here too

Reply:
Where did you find it?

Reply:
Can't find it anymore maybe my main branch was outdated when i copy/paste to here